README.rst

```diff
@@ -23,14 +23,16 @@ it... because paper.  I wrote this to make my life easier.
 How it Works
 ============
 
+Paperless does not control your scanner, it only helps you deal with what your 
+scanner produces
 1. Buy a document scanner like `this one`_ (used by me) or `this other one`_
```

Comment:
One last nitpick (sorry): you need a line break here, or you end up breaking the formatting.  Just look at the README on your branch in Github.  Once you fix this though, I'm happy to merge.